plugins/modules/win_acl_inheritance.ps1

```diff
@@ -56,9 +56,9 @@ function Remove-AclExplicitDuplicate {
 
     Process {
         $properties = $acl.Access |
-            Get-Member -MemberType Property |
-            Where-Object { $_.Name -ne 'IsInherited' } |
-            Select-Object -ExpandProperty Name
+        Get-Member -MemberType Property |
+        Where-Object { $_.Name -ne 'IsInherited' } |
+        Select-Object -ExpandProperty Name
```

Comment:
```suggestion
            Get-Member -MemberType Property |
            Where-Object { $_.Name -ne 'IsInherited' } |
            Select-Object -ExpandProperty Name
```
This change should be reverted in  my opinion, to avoid changing the git blame, and because specific style requirements are enforced in the repo. Indenting on the second and following lines of a multi-line pipeline is a common pattern.

Reply:
This is the reason for the sanity test failure btw

Reply:
this was inadvertent. i'll delete this PR and start over. too many issues.